lib/components/btn/s-btn.a11y.test.ts

```diff
@@ -1,5 +1,5 @@
-import { runComponentTests } from "./test-utils";
-import "../ts/index";
+import { runComponentTests } from "../../test/test-utils";
+import "../../index";
 
 describe("s-btn", () => {
```

Comment:
I just realized that we named this directory `btn` (and similarly `btn-group`). I understand that the abbreviation is what we use in the css classes but my opinion is that we should not tight the API naming we use necessarily to the file names. If you look at the razor components or even the documentation website we spell out the entire name: button and button-group. Shall we do the same here?

Reply:
Good call. I'll make this change